include/fg/CPUCopy.hpp

```diff
@@ -0,0 +1,43 @@
+/*******************************************************
```

Comment:
We can have them in the fg namespace without keeping them in the fg folder.
I don't think these count as forge headers, ie. not included in include/forge.h. Hence I believe they should be outside.